src/EditorFeatures/CSharpTest/PdbSourceDocument/AbstractPdbSourceDocumentTests.cs

```diff
@@ -131,7 +131,7 @@ protected static async Task RunTestAsync(Func<string, Task> testRunner)
             var service = workspace.GetService<IMetadataAsSourceFileService>();
             try
             {
-                var file = await service.GetGeneratedFileAsync(project, symbol, signaturesOnly: false, allowDecompilation: false, CancellationToken.None).ConfigureAwait(false);
+                var file = await service.GetGeneratedFileAsync(project, symbol, signaturesOnly: false, MetadataAsSourceOptions.Default, CancellationToken.None).ConfigureAwait(false);
```

Comment:
>MetadataAsSourceOptions [](http://example.com/codeflow?start=103&length=23)

Chang in semantics? Previously false, now true (default).

Reply:
Yep, neither setting is used in the tests